react/react2/week/github-user/src/components/GitUserContext.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import { createContext } from "react";
+
+export const UserContext = createContext();
+
+export default function GitUserContext({ children }) {
+    const [user, setUser] = useState("");
```

Comment:
Only these states should be in the context, and the context should be in another file, Context.js, conventionally. Here you'll consume the context to fetch the data.

Reply:
I have created separate file for context and import to GitUserFetch file.